Update/wata_008.txt

```diff
@@ -3841,7 +3841,7 @@ void main()
 //　詩音が俺の腕に組み付き、想像通りのことを言い出した￥
 	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
 	OutputLine(NULL, "　詩音が俺の腕に組み付き、想像通りのことを言い出した。",
-		   NULL, " Shion said, predictably, linking her arm with mine.", Line_Normal);
+		   NULL, "Shion said, predictably, linking her arm with mine.", Line_Normal);
```

Comment:
It looks like the spacing at the beginning is intended to separate this clause from the preceding quote.

`"...Aren't you interested, though? Come on, let's take a peek," Shion said, predictably, linking her arm with mine.`

There are many examples of this in these recent changes that should be reverted.